src/biogeochem/CNGapMortalityMod.F90

```diff
@@ -216,12 +209,12 @@ subroutine CNGapMortality (bounds, num_soilp, filter_soilp, &
                am = min(1._r8, am + heatstress(p))
             else ! lpj didn't set this for grasses; cn does
                ! set the mortality rate based on annual rate
-               am = params_inst%am
+               am = r_mort(ivt(p))
```

Comment:
I saw that you removed "am" from line 37 above. That makes me wonder whether line 212 -- and the "am = min(..." three lines above -- are still used somewhere?

Reply:
That is a good point.  The variable am is within a use_cndv if block which is now deprecated. I don't think we have any CNDV tests anymore, but if we did I think we wouldn't get the same answers as before with my changes here because "m" would not longer be multiplied by "am". In theory we could remove that whole section of code.   But if we don't want to remove it for some reason.  I guess I could change the code to this, so that if use_cndv is invoked, then am will be used, as it was before.  But we don't have any CNDV testing to check this.

```
      do fp = 1,num_soilp
         p = filter_soilp(fp)

         if (use_cndv) then
            ! Stress mortality from lpj's subr Mortality.

            if (woody(ivt(p)) == 1._r8) then

               if (ivt(p) == 8) then
                  mort_max = 0.03_r8 ! BDT boreal
               else
                  mort_max = 0.01_r8 ! original value for all patches
               end if

               ! heatstress and greffic calculated in Establishment once/yr

               ! Mortality rate inversely related to growth efficiency
               ! (Prentice et al 1993)
               am = mort_max / (1._r8 + k_mort * greffic(p))

               ! Mortality rate inversely related to growth efficiency
               ! (Prentice et al 1993)
               am = mort_max / (1._r8 + k_mort * greffic(p))

               am = min(1._r8, am + heatstress(p))
            else ! lpj didn't set this for grasses; cn does
               ! set the mortality rate based on annual rate
               am = r_mort(ivt(p))
            end if

         else
               am = r_mort(ivt(p))
         end if

         m  = am/(get_average_days_per_year() * secspday)
```

Reply:
As far as I know, cndv has not worked (or not worked correctly, at least) for quite some time. So I would be fine with removing cndv code. I don't know if @ekluzek would agree.

Reply:
Thanks @slevis-lmwg .  I'm thinking now that maybe it wouldn't be a good idea to remove the cndv code piecemeal.  I think for now I'll make the code mod I suggested above.